.travis.yml

```diff
@@ -1,20 +1,27 @@
 sudo: required 
-language: generic 
-compiler:
-  - gcc
+language: generic
+dist: xenial
 notifications:
   email:
     on_success: change
     on_failure: always
     recipients:
       - ros-contributions@amazon.com
+      - travis-build@platform-notifications.robomaker.aws.a2z.com
```

Comment:
According to @juanrh this line caused this travis build failure https://travis-ci.org/aws-robotics/kinesisvideo-common/jobs/476253478. Let's wait to see what the resolution should be.

Reply:
Okay, looks like it's not a issue with `.travis.yml`. Thanks.